core/services/validators.go

```diff
@@ -121,6 +120,7 @@ func ValidateInitiator(i models.Initiator, j models.JobSpec, store *store.Store)
 
 func validateFluxMonitor(i models.Initiator, j models.JobSpec, store *store.Store) error {
 	fe := models.NewJSONAPIErrors()
+	minimumPollingInterval := models.Duration(store.Config.DefaultHTTPTimeout())
```

Comment:
Should we use the same formatting in our API and configuration for string durations?

Reply:
I'm not sure it would save us all that much. It seems like half the time we need `time.Duration` and the other half `models.Duration` so if we make the type in the schema `models.Duration` we could simplify the code here, but I think we'd complicate it elsewhere.

Reply:
Looks like we use the same format anyways and models.Duration would require an update.

It'd be best to use the same type for both, as it enforces consistency, but this should not hold back these changes.